File: Python/BankAccount.py
class BankAccount:
    ___dep_name = " "
    acno = " "
    actype = ""
    balance = 0.0

    def menu(self):
        print ("What do you want to do:")
        print ("1. Deposit")
        print ("2. Withdraw")
        print ("3. Display Balance")
        print ("4. Exit")
        ch = int(input ("Enter you choice:"))

        if ch==1:
            self.deposit()
        elif ch==2:
            self.withdraw()
        elif ch==3:
            self.display()
        elif ch==4:
            print("Thank you!")
        else:
            print ("Invalid option!")
            self.menu()

    def __init__(self):
        self.__dep_name = "Ritik"
        self.acno = 123
        self.actype = "SA"
        self.balance = 10000.00
    
    def deposit(self):
        amount = float(input("Enter amount to deposit: "))
        self.balance = self.balance + amount
        self.menu()

    def withdraw(self):
        amount =  float(input("Enter amount to withdraw: "))
        if amount <= self.balance:
            self.balance = self.balance - amount
        else:
            print ("Transaction Declined!")
        self.menu()

    def display(self):
        print ("Account #: ",self.acno)
        print ("Depositor Name: ",self.__dep_name)
        print ("Account Type: ",self.actype)
        print ("Balance: ",self.balance)
        self.menu()

acc = BankAccount()
#acc.init()
#acc.__dep_name = "Banty Kumar"     Need more citation
acc.menu()

File: PHP/Console/BankAccount.php
<?php

class BankAccount
{
    private $dep_name;
    private $acno;
    private $actype;
    private $balance;

    function menu()
    {
        echo ("What do you want to do:\n");
        echo ("1. Deposit\n");
        echo ("2. Withdraw\n");
        echo ("3. Display Balance\n");
        echo ("4. Exit\n");
        echo ("Enter you choice:");
        $ch = readline();
        switch ($ch) {
            case 1:
                $this->deposit();
                break;
            case 2:
                $this->withdraw();
                break;
            case 3:
                $this->display();
                break;
            case 4:
                break;
            default:
                echo ("Invalid option!");
                $this->menu();
                break;
        }
    }

    function init()
    {
        $this->dep_name = "Ritik";
        $this->acno = 123;
        $this->actype = "SA";
        $this->balance = 10000.00;
    }

    function deposit()
    {
        echo ("Enter amount to deposit: ");
        $this->amount = readline();
        $this->balance = $this->balance + $this->amount;
        $this->menu();
    }

    function withdraw()
    {
        echo ("Enter amount to withdraw: ");
        $this->amount = readline();
        if ($this->amount <= $this->balance)
        $this->balance = $this->balance - $this->amount;
        else
            echo ("Transaction Declined!");
        $this->menu();
    }

    function display()
    {
        echo ("Account #: $this->acno\n");
        echo ("Depositor Name: $this->dep_name\n");
        echo ("Account Type: $this->actype\n");
        echo ("Balance: $this->balance\n");
        $this->menu();
    }
}


$acc = new BankAccount();
$acc->init();
$acc->menu();


//Scaffolding
